symphony/lib/toolkit/class.resourcespage.php

```diff
@@ -347,7 +349,29 @@ public function __actionIndex($resource_type)
                         // Don't allow Extension resources to be deleted. RE: #2027
                         if (stripos($path, EXTENSIONS) === 0) {
                             continue;
-                        } elseif (!General::deleteFile($path)) {
+                        }
+                        
+                        /**
+                         * Prior to deleting the Resource file. Target file path is provided.
+                         *
```

Comment:
Any way we can add the `@delegate` in the comment block? I assume it's with this parameter that the online doc auto generate ?

Reply:
No since it is dynamic in this page. But we only need the `@delegate` bloc once and it's there already (in the content page)